images/registry/Dockerfile

```diff
@@ -6,6 +6,6 @@ ARG COCKPIT_RPM_URL
 ARG INSTALLER
 
 ADD . /container
-RUN /container/install.sh
+RUN /container/install.sh && mv /usr/share/cockpit/registry/manifest.disabled /usr/share/cockpit/registry/manifest.json
```

Comment:
install.sh is adding it's own manifest right now. This is probably better though, so we should replace that with this.